TAF_EasyRest/TestFramework/PageComponents/AdminPanelPageComponent.cs

```diff
@@ -8,30 +8,19 @@ public class AdminPanelPageComponent : BasePage
 
         private IWebElement _usersButton => driver.FindElement(By.XPath("//span[text() = 'Users']/ancestor::a"));
         private IWebElement _ownersButton => driver.FindElement(By.XPath("//span[text() = 'Owners']/ancestor::a"));
+        private IWebElement _moderatorsButton => driver.FindElement(By.XPath("//span[text() = 'Moderators']/ancestor::a"));
```

Comment:
your XPath is working well but can be simplified to just "//span[text() = 'Moderators']"

Reply:
Right!

Reply:
@LoveMyDaughter, I don't see that you've resolved Andrii's comment. Are you agree with him or not?

Reply:
I want to leave ancestor::a inside the XPath - it's more clear what element we are looking for: "a" is a hyperlink.